src/app/interfaces/query.ts

```diff
@@ -2,7 +2,9 @@ import * as querystring from 'querystring';
 
 export enum OrderBy {
   Name = 'name',
-  Date = 'date'
+  Date = 'date',
+  Downloads = 'downloads',
+  Rating = 'rating'
```

Comment:
Since this is global just double check this is not displaying on the browse page as well.

Reply:
done. only values that are shown in Browse are name and date sort values